color.h

```diff
@@ -46,6 +47,9 @@ int get_opaqueness(const color &c) noexcept;
 /// Get the redness
 int get_redness(const color &c) noexcept;
 
+/// Get random color using random number generator
+color get_random_respawn_color(std::mt19937& ran_num_gen) noexcept;

```

Comment:
I enjoy the name! It felt weird at first, but it is indeed the color a player gets when spawning.